utest.h

```diff
@@ -736,6 +741,12 @@ UTEST_WEAK int utest_main(int argc, const char *const argv[]) {
     } else if (0 ==
                utest_strncmp(argv[index], output_str, strlen(output_str))) {
       utest_state.output = utest_fopen(argv[index] + strlen(output_str), "w+");
+    } else if (0 == utest_strncmp(argv[index], list_str, strlen(list_str))) {
+      for (index = 0; index < utest_state.tests_length; ++index) {
+        puts(utest_state.tests[index].name);
```

Comment:
Could you use UTEST_PRINTF here?

Reply:
Didn't know about that function - thanks. Fixed